stats/tests/stats.t

```diff
@@ -0,0 +1,46 @@
+\l ml.q
+\l util/init.q
+\l stats/init.q
+
+np:.p.import[`numpy]
+ols:.p.import[`statsmodels.api]`:OLS
+wls:.p.import[`statsmodels.api]`:WLS
+
+plaintab:([]4 5 6.;1 2 3.;-1 -2 -3.;0.4 0.5 0.6)
+x:1000?1000
+xf:1000?100f
+plaintabn:plaintab,'([]x4:1 3 0n)
+plaintabn2:plaintab,'([]x4:`a`a`b)
+
+.ml.stats.percentile[x;0.75]~np[`:percentile][x;75]`
+.ml.stats.percentile[x;0.02]~np[`:percentile][x;2]`
+.ml.stats.percentile[xf;0.5]~np[`:percentile][xf;50]`
+.ml.stats.percentile[3 0n 4 4 0n 4 4 3 3 4;0.5]~3.5
+
+descKeys:`count`mean`std`min`q1`q2`q3`max
+.ml.stats.describeFuncs:descKeys!.ml.stats.describeFuncs[descKeys]
+("f"$flip value .ml.stats.describe[plaintab])~flip .ml.df2tab .p.import[`pandas][`:DataFrame.describe][.ml.tab2df[plaintab]]
+("f"$flip value .ml.stats.describe[plaintabn])~flip (.ml.df2tab .p.import[`pandas][`:DataFrame.describe][.ml.tab2df[plaintab]]),'"f"$([]x4:3 2,sdev[1 3 0n],1 0 1 2 3)
+all all(flip value .ml.stats.describe[plaintabn2])=flip (.ml.df2tab .p.import[`pandas][`:DataFrame.describe][.ml.tab2df[plaintab]]),'([]x4:3f,7#(::))
```

Comment:
The describe function should include tests for temporal type data. Additionally given the separate behaviour when only non numeric categorical data is provided i.e. `.ml.stats.describe[([]`a`b;"ab";"x"$til 2)]`